### DISCUSSION TITLE
How do you set a custom base URL?

### DISCUSSION THREAD

**Discussion by vuyp** [AntonOsika/gpt-engineer#742] [Q&A]
See title, can't figure it out since exporting OPENAI_API_BASE doesn't work (returns not found thing)

**Comment by sudheesh4** [comment]
You just need to change the base url in openai module. Something like this worked for me (using LM Studio):

import openai
openai.api_base = "http://localhost:1234/v1" # point to the local server
openai.api_key = "" # no need for an API key
